embassy-stm32/src/lib.rs

```diff
@@ -203,6 +215,10 @@ impl Default for Config {
             dma_interrupt_priority: Priority::P0,
             #[cfg(gpdma)]
             gpdma_interrupt_priority: Priority::P0,
+            #[cfg(ucpd)]
+            enable_ucpd1_dead_battery: false,
+            #[cfg(all(ucpd, stm32g0x1))]
```

Comment:
you can do `#[cfg(peri_ucpd1)]`, `#[cfg(peri_ucpd2)]` to query whether the current chip has a peripheral instance.

Reply:
oh, neat. where do these cfgs actually come from? couldn't figure it out

Reply:
grep for `rustc-cfg` in `build.rs`. This one in particular is here https://github.com/embassy-rs/embassy/blob/main/embassy-stm32/build.rs#L65